releasenotes/notes/release-2.0-3af0045e2261bd02.yaml

```diff
@@ -1,19 +1,20 @@
 ---
 prelude: >
   The Datadog APM Python team is happy to announce the release of v2.0.0 of ddtrace.
-  This release introduces a formal :ref:`versioning support policy<versioning_support_levels>` for the major
-  versions of the ddtrace library, drops support for Python 2.7, 3.5, and 3.6, and adds support for Python 3.12.
+  This release drops support for Python 2.7, 3.5, and 3.6. This release adds
+  support for Python 3.12.
 
 
   .. important::
-    ddtrace v2.0.0 drops support for Python 2.7, 3.5, and 3.6. Users running these unsupported
-    Python versions should consider upgrading their Python version or continue using ddtrace v1
-    of the ddtrace library, which is now in maintenance mode and will only receive bug fixes. 
 
-    Before upgrading to v2.0.0, we recommend users install
-    ``ddtrace>=1.18.0,<2.0.0`` and enable deprecation warnings. All removals to
-    the library interface and environment variables were deprecated on the 1.x
-    branch.
+    If you are on version of Python not supported by v2, we will continue to
+    maintain the ddtrace v1 with bug fixes.
+
+
+  .. note::
+    Before upgrading to v2.0.0, we recommend users install ``ddtrace~=1.18.0``
```

Comment:
Should this be `1.19.0` given that it's currently building for release?

Reply:
`1.18.0` should be enough for the relevant deprecations.